### ISSUE TITLE
sampling steps for Open Sora

### ISSUE THREAD

**Issue by LiewFeng** [NUS-HPC-AI-Lab/VideoSys#211]
There two places to set the sampling steps([1](https://github.com/NUS-HPC-AI-Lab/VideoSys/blob/master/videosys/pipelines/open_sora/pipeline_open_sora.py#L35), [2](https://github.com/NUS-HPC-AI-Lab/VideoSys/blob/master/videosys/pipelines/open_sora/pipeline_open_sora.py#L136)).  Only [this one](https://github.com/NUS-HPC-AI-Lab/VideoSys/blob/master/videosys/pipelines/open_sora/pipeline_open_sora.py#L136) is valid while the system  prints the invalid one.

`VideoSys - INFO: Init Pyramid Attention Broadcast. steps: 50. spatial broadcast: True, spatial range: 5, spatial threshold: [450, 930]. temporal    
                             broadcast: True, temporal range: 7, temporal_threshold: [450, 930]. cross broadcast: True, cross range: 9, cross threshold: [450, 930]. mlp         
                             broadcast: True.`

**Comment by oahzxl** [comment]
you are right. the timestep here (https://github.com/NUS-HPC-AI-Lab/VideoSys/blob/master/videosys/pipelines/open_sora/pipeline_open_sora.py#L35) is deprecated.
